dali/test/python/test_fw_iterators.py

```diff
@@ -125,12 +130,17 @@ def test_mxnet_iterator_last_batch_no_pad_last_batch():
     num_gpus = 1
     batch_size = 100
 
-    pipes, data_size = create_pipeline(lambda gpu: create_coco_pipeline(batch_size=batch_size, num_threads=4, shard_id=gpu, num_gpus=num_gpus,
-                                                                        data_paths=data_sets[0], random_shuffle=True, stick_to_shard=False,
-                                                                        shuffle_after_epoch=False, pad_last_batch=False), batch_size, num_gpus)
+    pipes, data_size = create_pipeline(
+        lambda gpu:
+        create_coco_pipeline(batch_size=batch_size, num_threads=4, shard_id=gpu,
+                             num_gpus=num_gpus, data_paths=data_sets[0],
+                             random_shuffle=True, stick_to_shard=False,
+                             shuffle_after_epoch=False, pad_last_batch=False), batch_size, num_gpus
+    )
```

Comment:
```suggestion
    pipes, data_size = create_pipeline(
        lambda gpu: create_coco_pipeline(batch_size=batch_size, num_threads=4, shard_id=gpu,
                                         num_gpus=num_gpus, data_paths=data_sets[0],
                                         random_shuffle=True, stick_to_shard=False,
                                         shuffle_after_epoch=False, pad_last_batch=False),
        batch_size, num_gpus)
```

Reply:
Github UI makes this one (and the next ones) really bad, but I feel this way is better. Please compare in your IDE. I'm referring to putting `batch_size` and `num_gpus` at the indentation point of the lambda. This way it's clearer, these are args to `create_pipeline`.
![image](https://user-images.githubusercontent.com/7101064/176634503-ae4db12b-dd8f-47eb-bb3a-bc050beb3b7b.png)